### ISSUE TITLE
weird message on deploy

### ISSUE THREAD

**Issue by dennismckinnon** [eris-ltd/epm-go#148]
So in prep for tomorrow i was trying to break epm again. I'm on the develop branch and i tried deploying gendoug.sol onto a thel chain. I expected a vm error or something instead i got the following:

```
dennis@Juzo:~/Documents/go/src/github.com/eris-ltd/eris-std-lib/solidity$ epm cmd deploy gendoug.sol {{gendoug}}
2015/04/16 21:47:07 compiling remotely... http://ps.erisindustries.com:8090/compile
2015/04/16 21:47:07 [EPM] Executing job: deploy
2015/04/16 21:47:07 [SERV] Invalid chainId 49aa20cbeb2c1bf4a5d0863f1ac2e61b04cde50c. Disabling [eth]
2015/04/16 21:47:08 [SERV] Invalid chainId 49aa20cbeb2c1bf4a5d0863f1ac2e61b04cde50c. Disabling [eth]
Storing: gendoug 8665d28d83238c222c08fc5abbc50213a586c4b4
2015/04/16 21:47:08 [EPM] Deployed 8665d28d83238c222c08fc5abbc50213a586c4b4 as gendoug
dennis@Juzo:~/Documents/go/src/github.com/eris-ltd/eris-std-lib/solidity$ epm accounts
388f2d55294a17d777e070d463027d5abfbdc5f7 account
bbbd0256041f7aed3ce278c56ee61492de96d001 account
dennis@Juzo:~/Documents/go/src/github.com/eris-ltd/eris-std-lib/solidity$ epm plop addr
388f2d55294a17d777e070d463027d5abfbdc5f7
dennis@Juzo:~/Documents/go/src/github.com/eris-ltd/eris-std-lib/solidity$ epm head
thelonious/d85fa96bd942328710b726be1d89d86724464eaf
dennis@Juzo:~/Documents/go/src/github.com/eris-ltd/eris-std-lib/solidity$ epm refs
Name:               Blockchain:                                                 Address:            
bob                 thelonious/d85fa96bd942328710b726be1d89d86724464eaf         0x388f2d55294a17d777e070d463027d5abfbdc5f7
eth                 ethereum/default                                            0x6f30637867d78b50ad11403be2c8ab6e2d1fc770
joe                 thelonious/49aa20cbeb2c1bf4a5d0863f1ac2e61b04cde50c         0xdd6d3a69b3e438454965208bac750a10e67a6385
test                thelonious/a3be5e55c672a31ae4a84060b9d28c5cfd6de3eb         0x40fea8bb8c4c8e8d4f0bc293f0533e13ec1cc589
```

the Invalid chainId messages i don't understand because thats not even the chain checked out. this is probably just a missunderstand of the error message and not an actual bug. but i record it here just in case.

**Comment by ebuchman** [comment]
do you by change have `use_seed` on in the config? this is in response to a status message from a peer.

**Comment by dennismckinnon** [comment]
hmm

Its interesting that its a peer message because just before this I had fetched the 49aa chain from our 2gather peer server and then I switched to bob and ran that chain.

Both of those are thelonious chains. Does thelonious store peers on a per-chain basis? or is it globally cached somewhere? If its a global cache then it makes sense that when i switched chains an ran again it would throw an invalid chainid

To answer your question joe (the 49aa chain) does have the use_seed option but bob (chain i was on at the time) does not.
